src/init.ts

```diff
@@ -79,14 +87,14 @@ export class NightwatchInit {
       // Copy examples
       // For cucumber, only copy the cucumber examples.
       // For rest, copy all examples but cucumber.
-      if (answers.runner === 'cucumber') {
+      if (answers.runner === Runner.Cucumber) {
         this.copyCucumberExamples(answers.examplesLocation || '');
       } else if (answers.addExamples) {
         this.copyExamples(answers.examplesLocation || '', answers.language === 'ts');
         
         // For now the templates added only for JS
         if (answers.language !== 'ts') {
-          this.copyTemplates(answers.examplesLocation || '');
+          this.copyTemplates(path.join(answers.examplesLocation || '', EXAMPLE_TEST_FOLDER));
```

Comment:
We cannot put the templates inside `EXAMPLE_TEST_FOLDER` because we want the users to be able to run the example tests directly after the setup and putting templates there would result in error or failure of tests as template tests are incomplete.

Reply:
I am not sure if we should keep those templates. @beatfactor what do you think?

Reply:
It doesn't matter that much where we put them. The important thing is that if users go into the tests folder and delete everything, the test run should not throw errors.

Reply:
Creating a new issue for this.